src/types/UiaTextRangeBase.cpp

```diff
@@ -293,7 +276,7 @@ void UiaTextRangeBase::_expandToEnclosingUnit(TextUnit unit)
     // If we're past document end,
     // set us to ONE BEFORE the document end.
     // This allows us to expand properly.
-    if (bufferSize.CompareInBounds(_start, documentEnd, true) >= 0)

```

Comment:
FYI: The final boolean parameter in `CompareInBounds` is ignored in Release mode builds anyways, so from a perspective of "could this potentially crash?" it doesn't matter if the old code used to have `true`, `false`, or no boolean argument at all.